regression/python/type-annotation-assign-check/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.py
+--overflow-check
```

Comment:
Why do we need to enable --overflow-check? I suggest removing this dependency. The isinstance assertion should detect the type mismatch.